src/main/java/uk/gov/pay/adminusers/model/PSPTestAccountStage.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.pay.adminusers.model;
+
+public enum PSPTestAccountStage {
```

Comment:
Unsure if we want to use all CAPS for PSP. Any issue if we use camel case `PspTestAccountStage` ?